sdks/python/test-suites/portable/py2/build.gradle

```diff
@@ -147,10 +148,10 @@ task crossLanguagePortableWordCount {
   }
 }
 
-task postCommitIT {
+task postCommitPy2IT {
   dependsOn 'installGcpTest'
   dependsOn 'setupVirtualenv'
-  dependsOn ':runners:flink:1.8:job-server:shadowJar'
```

Comment:
I don't know much about these tests. Why did you upgrade this?

Reply:
This is a task that executes BigQuery read IT tests on Flink. It was added by me in the previous PR. I upgraded this to keep it consistent with Py 3.x versions:

https://github.com/apache/beam/blob/166ed817ab11f1d92ab2b6e0127673be5d0ea9c3/sdks/python/test-suites/portable/py37/build.gradle#L47-L50

Also, I included this task in the PostCommit suite (I accidentally hadn't done this earlier)